backend/requirements.txt

```diff
@@ -11,7 +11,7 @@ djangorestframework-simplejwt==5.2.0
 drf-spectacular==0.23.1
 drf-nested-routers==0.93.4
 pyjwt==2.6.0
-ipython==8.4.0
+ipython==8.8.0
 ipython-genutils==0.2.0
```

Comment:
```suggestion
```

These dependency file was created 4 years ago... Maybe these are not up to date at all... We can try to remove them and launch the tests ?

Reply:
@guilhem-barthes